.github/workflows/build-and-push-assets.yml

```
@@ -280,23 +280,23 @@ jobs:
           git commit -m "[BOT] Add compiled assets for #${{ github.ref }}" --no-verify || ((echo "NO_CHANGES=yes" >> $GITHUB_ENV) && (echo "No changes to commit"))
```

The `NO_CHANGES` env variable is no longer used, so we don't have to populate it here (untested!).
```suggestion
          git commit -m "[BOT] Add compiled assets for #${{ github.ref }}" --no-verify || echo "No changes to commit"
```
Please also remove it from line 148 and remove the unused `LOCK_FILE` env variable according to the Boy Scout rules. Thanks!

---

Obsolete with the latest amendments.